addons/knowledge/static/src/scss/knowledge_views.scss

```diff
@@ -198,6 +198,12 @@
     .o_knowledge_form_view {
         .o_scroll_view {
             @include o-position-absolute(0, 0, 0, 0);
+            h1, h2, h3, h4, h5, h6 {
+                &[highlight="true"] {
+                    background-color: $o-gray-300;
+                    transition: background-color 0.5s ease;
+                }
+            }
```

Comment:
same here, should be in common css file

Reply:
Is there already one that exists? (I don't think so?)

Otherwise, is it worth creating one for just this rule?